ogr/services/gitlab/pull_request.py

```diff
@@ -93,6 +94,15 @@ def labels(self) -> List[str]:
     def diff_url(self) -> str:
         return f"{self._raw_pr.web_url}/diffs"
 
+    @property
+    def source_project(self) -> "ogr_gitlab.GitlabProject":
+        if self._source_project is None:
+            source_project_id = self._raw_pr.attributes["source_project_id"]
+            self._source_project = self._target_project.service.get_project(
+                iid=source_project_id
+            )
+        return self._source_project
```

Comment:
Is it possible to use the same approach as you use for GitHub and avoid changing the `get_project` method?